src/App/Docs.hs

```diff
@@ -0,0 +1,19 @@
+module App.Docs (
+  userGuideUrl,
+  newIssueUrl,
+) where
+
+import App.Version (versionNumber)
+import Data.Text (Text)
+
+sourceCodeUrl :: Text
+sourceCodeUrl = "https://github.com/fossas/spectrometer"
+
+userGuideUrl :: Text
+userGuideUrl = guidePathOf $ maybe "master" ("v" <>) versionNumber
```

Comment:
I kept `master` as default revision, to ensure tests do not fail in dev workflow. Since current branch, or commit may not have been pushed to origin. Furthermore, we only "release" tagged versions anyways.